src/wallet/walletdb.cpp

```diff
@@ -1047,15 +1055,8 @@ bool AttemptBackupWallet(const CWallet& wallet, const fs::path& pathSrc, const f
         fs::copy_file(pathSrc.c_str(), pathDest, fs::copy_options::overwrite_existing);
 #elif BOOST_VERSION >= 105800 /* BOOST_LIB_VERSION 1_58 */
         fs::copy_file(pathSrc.c_str(), pathDest, fs::copy_option::overwrite_if_exists);
-#else
-        std::ifstream src(pathSrc.c_str(),  std::ios::binary | std::ios::in);
-        std::ofstream dst(pathDest.c_str(), std::ios::binary | std::ios::out | std::ios::trunc);
-        dst << src.rdbuf();
-        dst.flush();
-        src.close();
-        dst.close();
```

Comment:
Removing this fallback block breaks compatibility with Boost 1.57, which is our current minimum supported version. so it should probably be left in place until we decide to bump the minimum boost version.

Reply:
done